components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/utils/APIImportUtil.java

```diff
@@ -313,7 +313,7 @@ public static void importAPI(String pathToArchive, String currentUser, boolean i
                 // (Adding missing attributes to swagger)
                 SwaggerData swaggerData = new SwaggerData(importedApi);
                 String newDefinition = apiDefinition.generateAPIDefinition(swaggerData, swaggerContent);
-                apiProvider.saveSwaggerDefinition(importedApi, newDefinition);
+                apiProvider.saveSwaggerDefinition(importedApi, newDefinition, null);
```

Comment:
Lets try to pass tenant domain value

Reply:
https://github.com/wso2/carbon-apimgt/pull/9658/commits/142cd9cad67662fc64ac6e886127be649322fd43